onnxruntime/core/optimizer/selectors_actions/selector_action_transformer.cc

```diff
@@ -93,20 +93,15 @@ static Status MatchAndProcess(
   Status status = Status::OK();
 
   do {
-    // TODO: for now this just needs to support ONNX and Micrsoft Domain ops.
-    // If we ever had a transformer that was going to target non-ONNX ops,
-    // we'd need to rework a few things to include the op domain in the matches
-    if (node.Domain() != kOnnxDomain && node.Domain() != kMSDomain) {
-      break;
-    }
-
     std::optional<NodesToOptimizeIndices> node_selection_opt{};
     const SelectorActionRegistry::Entry* selector_action_entry_ptr = nullptr;
 
-    const auto selector_action_entries = selector_action_registry.LookUpByOpType(node.OpType());
+    const auto selector_action_entries = selector_action_registry.LookUpByOpTypeAndDomain(node.OpType(), node.Domain());
+    const auto& domain = node.Domain();
+    std::string domain_optype = (domain == kOnnxDomain) ? node.OpType() : (domain + ":" + node.OpType());
```

Comment:
maybe a helper function would be useful. e.g.,
```c++
std::string OpVersionsMapKey(std::string_view op_type, std::string_view domain = kOnnxDomain) {
  return (domain == kOnnxDomain)
      ? std::string{op_type}
      : std::string{domain} + ":" + std::string{op_type};
}
```
it could be used here and by callers of SelectorActionTransformer::RegisterSelectorAndAction. the callers wouldn't need to know the details of how to construct the key.

Reply:
Done